gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -164,13 +161,20 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         {%- endif %}
         """
         # Create or coerce a protobuf request object.
-        {% if method.legacy_flattened_fields -%}
+        {% if method.flattened_fields -%}
+        # Sanity check: If we got a request object, we should *not* have
+        # gotten any keyword arguments that map to the request.
```

Comment:
Do you intend to render this comment in generated gapic client?

Reply:
Yes, this comment is intended to be rendered in the generated client logic.